### PR TITLE
ACM-5066 pool-destruction-status-is-inconsist

### PR DESCRIPTION
**Pull request by jeswanke**
Signed-off-by: John Swanke <jswanke@redhat.com>

### CODE REVIEW

**Review by KevinFCormier — CHANGES_REQUESTED** [review]
@jeswanke Can you please use the `AcmInlineStatus` component? In other places we do not use a `Label`.

**Comment by jeswanke** [comment]
@KevinFCormier done

File: frontend/src/routes/Infrastructure/Clusters/ClusterPools/ClusterPools.tsx [hunk 1]
```diff
@@ -290,11 +291,7 @@ export function ClusterPoolsTable(props: {
             header: t('table.cluster.statuses'),
             cell: (clusterPool: ClusterPool) => {
               if (isClusterPoolDeleting(clusterPool)) {
-                return (
-                  <Label color="grey" icon={<TrashIcon />}>
-                    {t('Deleting')}
-                  </Label>
-                )
+                return <AcmInlineStatus type={StatusType.progress} status={t('Destroying')} />
```
Comment (KevinFCormier) [inline frontend/src/routes/Infrastructure/Clusters/ClusterPools/ClusterPools.tsx:294]:
```suggestion
                return <AcmInlineStatus type={StatusType.progress} status={t('destroying')} />
```

For some reason the translation key starts with lower case.

**Pull request merged**